### DISCUSSION TITLE
Not able to see download as pivoted table from pivot table

### DISCUSSION THREAD

**Discussion by manojrustagi79** [apache/superset#20791] [General]
I am using Apache Superset 2.0.0 on Cent OS 7 and Python 3.8.13. I have created a pivot table using my data set but when I am trying to download the PIVOT table then I am not able to see the download as PIVOTED table. 

Also in the same version I have configured email details for dashboard but it is showing on front end.

Please suggest what can I do on this.

**Comment by rusackas** [comment]
Closing this thread in favor of the same discussion happening here: https://github.com/apache/superset/discussions/19705
